src/configuration.js

```diff
@@ -150,6 +155,7 @@ export default class Configuration {
      * @param {Object} [options.logger] - a logger (see ./_flow/logger.js)
      * @param {Object} [options.tags] - set of key-value pairs which will be set
      *        as process-level tags on the Tracer itself.
+     * @param {Object} [options.upsampling] - overrides upsampling set on config
```

Comment:
this should be passed via config. `options` here is typically for things that cannot be initialized from a config file, like live dependencies.